src/cljs/rems/administration/blacklist.cljs

```diff
@@ -26,6 +26,27 @@
              :error-handler (flash-message/default-error-handler :top description)}))
    {:db (assoc db ::loading? true)}))
 
+(rf/reg-event-fx
+ ::remove-from-blacklist
+ (fn [{:keys [db]} [_ resource user]]
+   (let [description [text :t.administration/remove]]
+     (post! "/api/blacklist/remove"
+            {:params {:blacklist/resource (select-keys resource [:resource/ext-id])
+                      :blacklist/user (select-keys user [:userid])
+                      :comment ""}
```

Comment:
since the blacklist is such a serious thing, perhaps we should support comments in removals too

Reply:
Ok. I'll add it in the follow up PR.